WTWSMS/common/religions/00_religions.txt

```diff
@@ -1259,7 +1265,14 @@ muslim = {
 		landed_kin_prestige_bonus = yes
 		attacking_same_religion_piety_loss = yes
 		uses_decadence = no
-		uses_jizya_tax = yes
+		uses_jizya_tax = no
+		can_hold_temples = no
```

Comment:
What do you mean? I hope that you are aware that the wording "kebab" to refer to Turks is considered as racial slur and not allowed on the Pi forums?